File: 프로그래머스/lv0/120853. 컨트롤 제트/컨트롤 제트.js
function solution(s) {
    const strArr = s.split(' ');
    strArr.filter((el, idx) => el === 'Z' ? strArr[idx - 1] = '' : el)
    return strArr.reduce((acc, cur) => cur !== 'Z' ? acc + Number(cur) : acc, 0)
}

File: 프로그래머스/lv0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    strList = s.split(' ');
    for i in range(len(strList)):
        if strList[i] == 'Z':
            strList[i - 1] = 0
            strList[i] = 0
        else: strList[i] = int(strList[i])
    return sum(strList)